microservice/frontend/catalog/debian/data/etc/nginx/sites-available/shop.microservice.io.1

```diff
@@ -0,0 +1,14 @@
+server {
+  listen   81; ## listen for ipv4; this line is default and implied
+  #listen   [::]:80 default ipv6only=on; ## listen for ipv6
+
+  root /usr/share/shop_A/frontend/catalog/html;
```

Comment:
@ojacques working with @chavezom to see if we can fix Jenkins but we still do not completly understand your comment to add same *copy into*